spec/features/signed_login_flow_spec.rb

```diff
@@ -178,16 +178,16 @@
 
     it 'requires email validation when edited' do
       arrive_from_app(params: signed_params, do_expect: false)
-      fill_in (t :"signup.start.email_placeholder"), with: 'test-modified@test.com'
+      fill_in (t :"signup.start.email_placeholder"), with: 'test-modified@example.com'
```

Comment:
Reviewer can pretty much ignore this type of change. `mailcheck`er was showing the likely mistype warning for the domain `@test.com`. Should be no big deal.